Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM php:7.1
+FROM php:7.2
```

Comment:
7.4? nebo to nejde? aspon bych na to zalozil jestli to nejde hned

Reply:
ok, zalozim task, chcel som ist cestou najmensieho rizika.
https://keboola.atlassian.net/browse/COM-554